src/screens/buy/disclaimer/disclaimer.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { Divider } from '../../../components/divider/divider';
+import { Icon } from '../../../components/icon/icon';
+import { IconNameEnum } from '../../../components/icon/icon-name.enum';
+import { formatSize } from '../../../styles/format-size';
+import { useDisclaimerStyles } from './disclaimer.styles';
+
+export const Disclaimer = () => {
```

Comment:
could you reuse the existing component? the same component already exist when you do reveal secrets